Godeps/Godeps.json

```diff
@@ -34,8 +33,8 @@
 		},
 		{
 			"ImportPath": "github.com/opencontainers/runtime-spec/specs-go",
-			"Comment": "v1.0.0-rc1-31-gbb6925e",
-			"Rev": "bb6925ea99f0e366a3f7d1c975f6577475ca25f0"
+			"Comment": "v1.0.0-rc5-30-g3adac26",
```

Comment:
Is this intentionally bumping to a commit that is 30 commits after the tagged rc5?  I don't think the `LinuxSyscall.Comment` removal or `Linux.IntelRdt` additions are so pressing that it's worth bringing in a non-release spec commit.